benchmarks/dynamo/microbenchmarks/inductor_cpu_atomic.py

```diff
@@ -71,8 +71,8 @@ def test_total_time(shapes, types):
 """
 test total time
 shape; type; torch scatter_add; inductor scatter_add; torch scatter_add (worst case); inductor scatter_add (worst case)
-[1, 4096]; torch.float32; 0.14733232000025964; 0.05388864999986254; 0.1451428800010035; 0.06496850000075938
-[1, 4096]; torch.int32; 0.1440268700002889; 0.05882900999949925; 0.1429359899998417; 0.07036211000013282
-[1, 65536]; torch.float32; 1.3435545300012564; 0.15207924000151252; 1.2523296799986383; 3.1408327299982375
-[1, 65536]; torch.int32; 1.3407247500003905; 0.12999147000073208; 1.2956029100018895; 0.853825209999286
+[1, 4096]; torch.float32; 0.0052365999999892665; 0.017564740001034806; 0.013978460001453641; 0.024368539998249616
+[1, 4096]; torch.int32; 0.005814169999212027; 0.016935479998210212; 0.005414689994722721; 0.017164559994853335
+[1, 65536]; torch.float32; 0.1297304199943028; 0.13822156000060204; 0.24826899999425223; 3.4930834800070443
+[1, 65536]; torch.int32; 0.126586229998793; 0.1042632999997295; 0.10594573999696877; 1.0463533599977382
```

Comment:
Do you mean the perf speedup of eager is from the clang compiler you were using? What compiler did you use for inductor codegen? Just curious why does the perf numbers differ so much. Inductor's perf is much slower than eager for small shapes. I guess we'd better understand the gap first. cc @zhuhaozhe

Reply:
@jgong5 Sorry, after some investigation I found that I had the lower speed of eager due to the debug build in my last commit ( :joy: I have no idea when I have set the DEBUG env last time)
So the perf speedup is due to Release vs Debug build.